custom_model_runner/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Show full R traceback for fit/predict errors
 ##### Fixed
 - Loading hooks from both custom.R and custom.r
+- Using -cd instead of --code-dir failed sometimes when running with docker
```

Comment:
maybe better to be more official here. sometimes -> occasionally

Reply:
Will fix in the release PR. Should remove 'sometimes' because it happens every time